slapp/static/config/parameters_slider.json

```diff
@@ -0,0 +1,94 @@
+{
+  "1": {
+    "pv": {
+      "class": "js-range-slider",
+      "max": 0.95,
+      "min": 0,
+      "from": 0.12,
+      "step": 0.01
+    },
+    "wind": {
+      "class": "js-range-slider",
+      "max": 1.1,
+      "min": 0,
+      "from": 0.1,
+      "step": 0.01
+    },
+    "hydro": {
+      "class": "js-range-slider",
+      "max": 0.8,
+      "min": 0,
+      "from": 0.18,
+      "step": 0.01
+    }
```

Comment:
biomass and energy storage are missing - doe we need them?
If so, please add them here for example muns and check if sliders are working on ESM config page.
Otherwise, please remove biomass and energy storage config from config page.

Reply:
In the corresponding issue it was only saying: " implement sliders for wind, PV roof, PV ground and hydro" but maybe they were only not needed then? 

> biomass and energy storage are missing - doe we need them? If so, please add them here for example muns and check if sliders are working on ESM config page. Otherwise, please remove biomass and energy storage config from config page.

Reply:
@nesnoj do you know if biomass and energy storage are needed in the parameters? And if so if they also should have sliders as an input value?

Reply:
yes, please